templates/__common__/config/tasks/graphics-meta.js

```diff
@@ -172,6 +178,15 @@ const parseGraphic = async (
     outputPath,
   });
 
+  // check if default tags are used
```

Comment:
nice I like this warning message 👍 it's something we've been wanting to add to the coronavirus tracker